lib/send-mail.ts

```diff
@@ -53,7 +53,8 @@ export async function parseMBox(mbox: string, gentle?: boolean): Promise<IParsed
     const parsed = await simpleParser(mbox, options);
 
     for (const entry of parsed.headerLines) {
-        const valueSet = entry.line.match(/(.*?): *([^]*)$/);
+        // try to parse header line and consume a leading line break after the colon in folded headers
+        const valueSet = entry.line.match(/(.*?):(?:\r?\n)? *([^]*)$/);
```

Comment:
I think this will work.

While looking at this regex, I cannot fail to notice that it isn't high quality (probably my work, honestly): Header keys ["MUST be composed of printable US-ASCII characters (i.e., characters that have values between 33 and 126, inclusive), except colon"](https://datatracker.ietf.org/doc/html/rfc2822#section-2.2), and they must be at the start of the line. Therefore, I think we will want to consider using `/^([^ :]+):` instead of `/(.*?):`, but that's outside _this PR_'s scope.